firestore/api/FirestoreTest/FirestoreTest.cs

```diff
@@ -829,5 +833,14 @@ public void MultipleCursorConditionsTest()
             RunQueryData("query-create-examples", Environment.GetEnvironmentVariable("FIRESTORE_PROJECT_ID"));
             RunPaginateData("multiple-cursor-conditions", Environment.GetEnvironmentVariable("FIRESTORE_PROJECT_ID"));
         }
+
+        [Fact]
+        public void RunDistributedCounterTest()
+        {
+            var output = RunDistributedCounter("distributed-counter", Environment.GetEnvironmentVariable("FIRESTORE_PROJECT_ID"));
+            Assert.Contains("Distributed counter created.", output.Stdout);
+            Assert.Contains("Distributed counter incremented.", output.Stdout);
+            Assert.DoesNotContain("Total count: 0", output.Stdout);
```

Comment:
Should be 1, right?
Maybe better `Assert.Contains("Total count: 1", output.Stdout);`